seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/AbstractSeleniumJSRunner.scala

```diff
@@ -28,6 +28,13 @@ abstract class AbstractSeleniumJSRunner(browserProvider: SeleniumBrowser,
     _console = console
   }
 
+  protected def ignoreKeepAlive: Boolean = {
+    val name = code.name
+    name == "frameworkDetector.js" ||
+    name == "testFrameworkInfo.js" ||
+    name == "testMaster.js"
```

Comment:
Hum ... That's unfortunate.
Let's keep it that way for now. But we should think whether we can do something more principled in the future.